core/src/main/java/org/axonframework/eventsourcing/eventstore/TrackingToken.java

```diff
@@ -16,12 +16,17 @@
 
 package org.axonframework.eventsourcing.eventstore;
 
+import com.fasterxml.jackson.annotation.JsonTypeInfo;
+
 /**
  * Tag interface identifying a token that is used to identify the position of an event in an event stream. Event
  * processors use this token to keep track of the events they have processed and still need to process.
  *
  * @author Rene de Waele
  */
+@JsonTypeInfo(use = JsonTypeInfo.Id.CLASS,
```

Comment:
I assume this guy is required for correct deserialization of the `ReplayToken`, as that guy contains two `TrackingTokens`, correct?

Reply:
Is it possible to configure this in the `ReplayToken` instead? I'm always a bit reluctant to have these kind of generic changes that may affect other things than just the issue at hand...

Reply:
Yes it is! Change is on the way ;)